de.metas.adempiere.libero.libero/src/main/java/org/eevolution/model/validator/PP_Product_BOMLine.java

```diff
@@ -127,4 +114,14 @@ public void updateProductLowestLevelCode(final I_PP_Product_BOMLine bomLine)
 
 		// TODO: if product's low level changed, we need to update the low level from all bom components where this product is in bom header.
 	}
+
+	@CalloutMethod(columnNames = { I_PP_Product_BOMLine.COLUMNNAME_VariantGroup})
```

Comment:
Tehre already is a callout class PP_Product_BOMLine.
Please move this callout code there. Alternatively, move all the PP_Product_BOMLine callout code here